recipes-hosted-login/mender-hosted-login/mender-hostedlogin-0.1/menderLogin.py

```diff
@@ -189,6 +190,15 @@ def _logIn_clicked(self):
                 f_tmp = open("/tmp/mender.conf.tmp", "w+")
                 # Rewrite mender.conf
                 json.dump(data, f_tmp, indent=2, sort_keys=True)
+
+                # Also write the tenant token to a persistent file
+                # this will be restored into mender.conf by the mender.service
+                # launch script.
+                p_tmp = open("/data/mender/hosted-mender-settings.dat", "w")
+                p_tmp.write(data["TenantToken"])
+                p_tmp.write("\n")
+                p_tmp.write(data["ServerURL"])
+                p_tmp.write("\n")
```

Comment:
Suggestion: 
```
json.dump({"ServerURL": data["ServerURL"], \
                   "TenantToken": data["TenantToken"] }, p_tmp)
```

Reply:
Yeah, that's definitely a better approach but for my current needs I think I'll keep it as simple as possible.